### ISSUE TITLE
Please add a method for returning the entire response header

### ISSUE THREAD

**Issue by johngluck65** [peritus/robotframework-httplibrary#4]
Right now, I can only look at a response by specific header name.  I want to be able to look at the entire response to know if there is, for example a duplicate header.  Right now I know way of doing that that I can see

**Comment by peritus** [comment]
Ok so, did a quick test:

Wrote a server that answers as follows when connected to w/ telnet (notice the duplicate header):

```
18:24:49 filip@nomanchompsky ~ telnet localhost 36503    
Trying ::1...
telnet: connect to address ::1: Connection refused
Trying 127.0.0.1...
Connected to localhost.
Escape character is '^]'.
GET /duplicate_header HTTP/1.1

HTTP/1.0 200 OK
Server: BaseHTTP/0.3 Python/2.7.1
Date: Sun, 08 Apr 2012 16:25:06 GMT
Duplicate: Yes
Duplicate: Yes

Connection closed by foreign host.
```

When write a test against that:

```
Test Duplicate Headers
    GET     /duplicate_header
```

and I run this w/ @robotframework's DEBUG loglevel like this:

```
rebot.py -L DEBUG tests/
```

the debug output in the HTML form looks like this:

![18:25:52.185  DEBUG   No request headers set
18:25:52.185    DEBUG   No request body set
18:25:52.185    DEBUG   Performing GET request on http://localhost:36503/duplicate_header
18:25:52.188    DEBUG   Response status line: 200 OK
18:25:52.188    DEBUG   Response headers:
18:25:52.188    DEBUG   date: Sun, 08 Apr 2012 16:25:52 GMT
18:25:52.189    DEBUG   duplicate: Yes, Yes
18:25:52.189    DEBUG   server: BaseHTTP/0.3 Python/2.7.1
18:25:52.189    DEBUG   No response body received](http://dl.dropbox.com/u/8735936/duplicate_header_debug.png)

.. does that work for you ? Because, otherwise, there might be a way to access the headers differently.

Or do you want to access (maybe duplicate headers) programmatically (and e.g. assert that there are no duplicate headers) ?

**Comment by johngluck65** [comment]
Yes, I'm trying to assert the negative, that there are no duplicate headers. 

Thanks for working this

**Comment by johngluck65** [comment]
Is there any progress on this.  I'd still like to be able to assure that there are no duplicates.

**Comment by peritus** [comment]
Fixed in v0.3.2 (released a few minutes ago). Please see https://github.com/peritus/robotframework-httplibrary/commit/f4c1a655886de83512683e704d2047dd0a79f803#diff-3 on how to use this.
